packages/main/src/plugin/extension/extension-analyzer.ts

```
@@ -124,9 +167,18 @@ export class ExtensionAnalyzer {
-    return JSON.parse(await readFile(manifestPath, 'utf8'));
+    const manifest = JSON.parse(await readFile(manifestPath, 'utf8'));
+
+    // try to parse using zod
+    const result = ExtensionManifestSchema.safeParse(manifest);
+    if (result.success) {
+      return result.data;
+    } else {
+      console.error(`Error while parsing manifest for extension ${extensionPath}. ${z.prettifyError(result.error)}`);
+      // return the manifest as it is, even if it is not valid
+      return manifest as ExtensionManifest;
+    }
```

Biome (2.4.4)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)